.github/workflows/deploy-pypi.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish to PyPI
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  build-and-test-sdist-and-wheels:
+    uses: glotzerlab/signac/.github/workflows/publish-packages.yml
```

Comment:
I think we want to use the action in the current repo state, rather than a "versioned" copy. This currently shows an error:

![image](https://user-images.githubusercontent.com/3943761/193980079-bdbd1eba-a789-443d-a295-258b9bb353e4.png)
https://github.com/glotzerlab/signac/actions/runs/3187228681/workflow

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses
```suggestion
    uses: ./.github/workflows/publish-packages.yml
```

Reply:
Yup, this was broken by 14ad15f4945f2a49cbd748ae3c87153009db00df.